src/pages/settings/AboutPage/ConsolePage.tsx

```diff
@@ -91,7 +97,7 @@ function ConsolePage({capturedLogs, shouldStoreLogs}: ConsolePageProps) {
                 return;
             }
 
-            Navigation.navigate(ROUTES.SETTINGS_SHARE_LOG.getRoute(path));
+            Navigation.navigate(ROUTES.SETTINGS_SHARE_LOG.getRoute(path, ROUTES.SETTINGS_CONSOLE.getRoute(route.params?.backTo)));
```

Comment:
Is there a case where we're navigated to `ROUTES.SETTINGS_SHARE_LOG` not from Console Page (`ROUTES.SETTINGS_CONSOLE`)?
If not and we can only go back to Console from Share Log page, than this change isn't needed (as well as the changes to `src/pages/settings/AboutPage/ShareLogList/BaseShareLogList.tsx`)